examples/thermomechanics/inputs/thermal_crack.json

```diff
@@ -0,0 +1,16 @@
+{
+    "num_cells"                     : {"value": [501, 101, 11]},
+    "system_size"                   : {"value": [0.05, 0.01, 0.001], "unit": "m"},
+    "density"                       : {"value": 3980, "unit": "kg/m^3"},
+    "elastic_modulus"               : {"value": 370e+9, "unit": "Pa"},
+    "fracture_energy"               : {"value": 24.32, "unit": "J/m^2"},
+    "thermal_coefficient"           : {"value": 7.5E-6, "unit": "oC^{-1}"},
+    "reference_temperature"         : {"value": 300.0, "unit": "oC"},
+    "background_temperature"        : {"value": 20.0, "unit": "oC"},
+    "background_temperature_ramp"   : {"value": 0.001, "unit": "s"},
```

Comment:
This is actually the "surface_temperature_ramp_time". 
First, the background temperature is unchanged. It's only the plate surface temperature.
Second, it is not temperature but time.

Reply:
Sure, as long as we're okay with this variable being very specific and not reused for other problem types